docs/micro-rdk/base/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Micro-RDK Base Models"
+linkTitle: "Base"
+weight: 30
+type: "docs"
+description: "The micro-RDK support for moving platforms that the other parts of a mobile robot attach to."
+images: ["/icons/components/base.svg"]
+tags: ["base", "components", "micro-rdk"]
+no_list: true
+# SMEs: Nick M., Gautham V.
+---
+
```

Comment:
[opt] I just realized this now so I realize it's late to bring up and could add in a later PR. But I'm wondering--should each of these component pages link to the corresponding RDK component pages for more info on what a base etc. is? Or at least have some information? For example
```suggestion

A base is the moving platform that other parts of a robot attach to.

By configuring a base component, organizing individual components to produce coordinated movement, you gain an interface to control the movement of the whole physical base of the robot without needing to send separate commands to individual motors.

```

Reply:
@JessamyT Thank you, that is a really good suggestion! Added abridged intros and linking to main docs for each component, LMK what you think.